vaccine_feed_ingest/stages/ingest.py

```diff
@@ -324,17 +326,40 @@ def _validate_normalized(output_dir: pathlib.Path) -> bool:
                     return False
 
                 if normalized_location.location:
-                    if not BOUNDING_BOX.latitude.contains(
-                        normalized_location.location.latitude
-                    ) or not BOUNDING_BOX.longitude.contains(
-                        normalized_location.location.longitude
-                    ):
+                    result = validate_bounding_boxes(
+                        normalized_location.location, APPROVED_BOUNDS
+                    )
+
+                    # if false, return false
+                    if not result:
                         logger.warning(
                             "Invalid latitude or longitude in %s at line %d: %s is outside approved bounds (%s)",
                             filepath,
                             line_no,
                             normalized_location.location,
-                            BOUNDING_BOX,
+                            APPROVED_BOUNDS,
                         )
                         return False
+
+    return True
+
+
+def validate_bounding_boxes(location, bounding_boxes):
+    results = []
+
+    for boundingbox in bounding_boxes:
```

Comment:
You can simplify this even more if you swap the boolean conditional and return True the first time you met it.

```
for bb in bbs:
  if bb.lat.contains() and bb.long.contains():
    return True

return False